js/stores/windowStore.js

```diff
@@ -750,6 +752,22 @@ const doAction = (action) => {
         }
         break
       }
+    case windowConstants.WINDOW_ON_TOGGLE_BOOKMARK:
+      {
+        // show the AddEditBookmarkHanger control; saving/deleting takes place there
+        const activeFrame = frameStateUtil.getActiveFrame(windowState)
+        let siteDetail = siteUtil.getDetailFromFrame(activeFrame, siteTags.BOOKMARK)
+        const key = siteUtil.getSiteKey(siteDetail)
+
+        if (key !== null) {
```

Comment:
we are only missing `parentFolderId` in this example, I just checked and everything else is there. But I think this whole PR, will be redundant when I do #9710 :smiley: